jvm-packages/CMakeLists.txt

```diff
@@ -14,6 +23,16 @@ target_include_directories(xgboost4j
   ${PROJECT_SOURCE_DIR}/dmlc-core/include
   ${PROJECT_SOURCE_DIR}/rabit/include)
 
+if (USE_CUDA)
```

Comment:
This is unnecessary as it's handled in top level CMake file.

Reply:
Done